dataland-community-manager/src/main/kotlin/db/migration/V2__GenerateCompanyNameIdentifierIndex.kt

```diff
@@ -0,0 +1,19 @@
+package db.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+
+/**
+ * This migration script creates an index for a faster search for company names and identifiers
```

Comment:
This that the correct description?

Reply:
@fsoffel Could you answer that? I'm personally not even sure why there are two different migration scripts - What is V1 doing if V2 is altering them? Looks like it creates the tables, why do we need that?